### DISCUSSION TITLE
Video & Audio unlinked when importing to Resolve

### DISCUSSION THREAD

**Discussion by MonzterDev** [WyattBlue/auto-editor#503] [4. Bugs]
The video and audio tracks are not linked together. I'm skipping the other questions because they're less relevant to this issue, which stems from this issue: https://github.com/WyattBlue/auto-editor/issues/472.

**Comment by WyattBlue** [comment]
Fixed a while ago?

**Comment by SchonKonnie** [comment]
I think he meant that not all audio tracks are linked if you have multiple streams.
Thats at least the case for me (version 25.0.1)

**Comment by WyattBlue** [comment]
> I think he meant that not all audio tracks are linked if you have multiple streams. Thats at least the case for me (version 25.0.1)

That is true

**Comment by WyattBlue** [comment]
See the discussion for DaVinci Resolve export notes/issues https://github.com/WyattBlue/auto-editor/discussions/504
